github-fixed-header.user.css

```diff
@@ -18,6 +18,9 @@
   .is-stuck {
     padding-top: var(--height) !important;
   }
+  body.logged-out {
+    padding-top: 89px !important;
```

Comment:
You meant `padding-top: -89px !important;` else it goes like

![capture](https://user-images.githubusercontent.com/31389848/53560319-66d06f00-3b4c-11e9-9d1a-098095cf1a85.PNG) here